test/definitions/checks/disk_performance_test.rb

```diff
@@ -1,11 +1,21 @@
 require 'test_helper'
-
 describe Checks::Disk::Performance do
   include DefinitionsTestHelper
   include UnitTestHelper
 
   let(:check_disk_performance) { described_class.new }
 
+  before do
+    assume_feature_absent(:mongo)
+    current_feature = if file_nonzero?('/var/lib/pulp')
+                        :pulp2
+                      else
+                        :pulpcore_database
```

Comment:
I don't like the way test failure is fixed instead of fixing the disk performance check.
@upadhyeammit, am I missing something here?

What I understood is that old pulp feature renamed to `pulp2` and confine block of this performance check 
```
confine do
   feature(:pulp)
end 
```
replaced with 
```

confine do
  feature(:instance).pulp
end
```
Inside feature(:instance), based on `pulp2` or `pulpcore` presence, pulp method returns the feature value. So I think instead of having `pulp` in default list at [line 36](https://github.com/theforeman/foreman_maintain/pull/287/files#diff-248e3e4ed2b45fee19280d48612afee0R36), we should actual add `pulp2` or `pulpcore_database` based on which pulp feature is available.